src/services/completions.ts

```diff
@@ -1925,28 +1925,28 @@ namespace ts.Completions {
                 preferences,
                 !!importCompletionNode,
                 context => {
-                    exportInfo.forEach(getChecker, (info, symbolName, isFromAmbientModule) => {
+                    exportInfo.forEach(sourceFile.path, (info, symbolName, isFromAmbientModule) => {
                         if (!detailsEntryId && isStringANonContextualKeyword(symbolName)) return;
                         const isCompletionDetailsMatch = detailsEntryId && some(info, i => detailsEntryId.source === stripQuotes(i.moduleSymbol.name));
                         if (isCompletionDetailsMatch || charactersFuzzyMatchInString(symbolName, lowerCaseTokenText)) {
-                            if (isFromAmbientModule && !some(info, isImportableExportInfo)) {
+                            const defaultExportInfo = find(info, isImportableExportInfo);
+                            if (isFromAmbientModule && !defaultExportInfo) {

```

Comment:
Are the two conditions related?  If not, how do we know `exportInfo` is defined below?

Reply:
I made a mental note to revisit this and see if I could break these conditions apart, but I forgot to at the time. It now makes more sense.